### PR TITLE
Fixed source code formatting.

### PR DESCRIPTION
**Pull request by thmiceli**
Ran tslint with some custom enhancements.

### CODE REVIEW

**Comment by ericvw** [comment]
Should `tslint.json` be updated as well as any dependency on `tslint`?

**Comment by thmiceli** [comment]
Since it may take a while before tslint incorporates my enhancements in a new release (assuming that they will take all of them). I can push here my custom tslint rules. I am not sure if I can override the existing rules that I modified.

**Comment by thmiceli** [comment]
Adding a dependency on the tslint project on github doesn't work because the output of a build is not pushed on github. I could however add a dependency to a fork of tslint that contains everything that is needed to run tslint out-of-the-box.

**Comment by ericvw** [comment]
I would advocate for asking in the PR to `tslint` when they plan to release a new `npm` module.  At least that is what I have done in the past.

**Comment by ericvw** [comment]
PR #110 is a prime candidate why it would be best to directly depend upon the `tslint` changes somehow :)

**Comment by thmiceli** [comment]
I added custom tslint rules and updated makefile so that they get built automatically.

File: tslint.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
     "rules": {
+        "align": [true, "parameters", "arguments", "statements"],
```
Comment (ericvw) [inline tslint.json]: Let's format this similarly to the other configuration options.  See `typedef` for example.

**Comment by ericvw** [comment]
lgtm - please squash

**Comment by ericvw** [comment]
Also, please update the commit message body to describe in detail the high-level changes that are a part of this change.

**Pull request merged**